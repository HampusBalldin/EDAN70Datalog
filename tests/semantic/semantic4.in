TYPEOF('A, [Integer String]).
B(x) :- A(x).
