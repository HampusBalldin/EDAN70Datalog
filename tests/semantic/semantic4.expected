Wrong number of arguments in: TYPEOF('A,[Integer String]), expected: 1
